modules/Auth/tests/auth_tests.cpp

```diff
@@ -385,16 +385,16 @@ TEST_F(AuthTest, test_transaction_finish) {
     SessionEvent session_event2;
     session_event2.event = SessionEventEnum::TransactionStarted;
     TransactionStarted transaction_event1;
-    transaction_event1.energy_Wh_import = 0;
-    transaction_event1.id_tag = provided_token_1.id_token;
+    transaction_event1.meter_value.energy_Wh_import.total = 0;
+    transaction_event1.id_tag = provided_token_1;
     transaction_event1.timestamp = Everest::Date::to_rfc3339(date::utc_clock::now());
     session_event2.transaction_started = transaction_event1;
 
     SessionEvent session_event3;
     session_event3.event = SessionEventEnum::TransactionStarted;
     TransactionStarted transaction_event2;
-    transaction_event2.energy_Wh_import = 0;
-    transaction_event2.id_tag = provided_token_2.id_token;
+    transaction_event2.meter_value.energy_Wh_import.total = 0;
+    transaction_event2.id_tag = provided_token_2;
     transaction_event2.timestamp = Everest::Date::to_rfc3339(date::utc_clock::now());
     session_event3.transaction_started = transaction_event2;
```

Comment:
Just a comment on this style, it is more readable if you would create a small helper function for this like
```static auto init_empty_transaction_event(const ProvidedIdToken& token) { // name of the function up to discussion
    TransactionStarted te;
    te.meter_value.energy_Wh_import.total = 0;
    te.id_tag = token; // this line doesn't make sense due to naming, how can you assign a token to an id_tag?
    te.timestamp = Everest::Date::to_rfc3339(data::utc_clock::now()); // there should be a helper function in Everest::Date for the 'now" case
    return te;
}
```

Now you can simply write:
```
session_event1.transaction_started = init_empty_transaction_event(provided_token_1);
session_event3.transaction_started = init_empty_transaction_event(provided_token_2);
```
and reduce LOCs